test/err.js

```diff
@@ -27,3 +28,16 @@ test('type error', function (t) {
         '{ [SyntaxError: whoa] message: \'whoa\', \'a-b\': 5 }'
     ].join(', ') + ' ]');
 });
+
+test('type AssertionError', function (t) {
+    t.plan(3);
+    var err = 'woot', obj2str = Object.prototype.toString;
```

Comment:
That's about debugger quality. If both expressions were in an array in one variable assignment, (`err = [ 'woot', Object.prototype.toString];`) I assume you'd expect your debugger to cope with it, too.